dali/dafilesrv/dafilesrv.cpp

```diff
@@ -691,6 +692,8 @@ int main(int argc,char **argv)
     } perfHook;
     startPerformanceMonitor(10*60*1000, PerfMonStandard, &perfHook);
     writeSentinelFile(sentinelFile);
+
+    setKeyIndexCacheSize(0); // disable IKeyIndex caching
```

Comment:
We don't generally overwrite files once written, what is the likely affect of disabling the cache?